stake-pool/js/src/index.ts

```diff
@@ -0,0 +1,167 @@
+import * as schema from './schema.js';
+import solanaWeb3 from '@solana/web3.js';
+import assert from 'assert';
+
+/**
+ * Sample code to demonstrate how to use the JS bindings
+ * Also contains several useful helper functions
+ */
+
+export class StakePoolAccounts {
+  /**
+   * Wrapper class for a stake pool.
+   * Each stake pool has a stake pool account and a validator list account.
+   */
+  stakePool: StakePoolAccount;
+  validatorList: ValidatorListAccount;
+}
+
+export interface StakePoolAccount {
+  pubkey: solanaWeb3.PublicKey;
+  account: solanaWeb3.AccountInfo<schema.StakePool>;
+}
+
+export interface ValidatorListAccount {
+  pubkey: solanaWeb3.PublicKey;
+  account: solanaWeb3.AccountInfo<schema.ValidatorList>;
+}
+
+export async function getStakePoolAccount(
+  connection: solanaWeb3.Connection,
+  stakePoolPubKey: solanaWeb3.PublicKey,
+): Promise<StakePoolAccount> {
+  /**
+   * Retrieves and deserializes a StakePool account using a web3js connection and the stake pool address.
+   * @param connection: An active web3js connection.
+   * @param stakePoolPubKey: The public key (address) of the stake pool account.
+   */
```

Comment:
I always thought these comments had to be before the code to apply correctly, e.g. 
```
/**
 * my comment
 */
export async function myFunction()
```
Can it also be in this position?